tests/backend_tests/test_asyncio.py

```diff
@@ -0,0 +1,25 @@
+from unittest.mock import MagicMock, patch
+
+import pytest
+
+from httpcore._backends.asyncio import SocketStream
+
+
+class TestSocketStream:
+    class TestIsReadable:
+        @pytest.mark.asyncio
+        async def test_returns_true_when_transport_has_no_socket(self):
+            stream_reader = MagicMock()
+            stream_reader._transport.get_extra_info.return_value = None
+            sock_stream = SocketStream(stream_reader, MagicMock())
+
+            assert sock_stream.is_readable()
+
+        @patch("httpcore._utils.is_socket_readable", MagicMock(return_value=True))
+        @pytest.mark.asyncio
+        async def test_returns_true_when_socket_is_readable(self):
+            stream_reader = MagicMock()
+            stream_reader._transport.get_extra_info.return_value = None
```

Comment:
Hmm. Should this be returning something truthy? Right now the two tests do the same thing, and the patched is_socket_readable helper isn't actually called.

Reply:
you're right, let me fix this :)